modules/indigo/module/inc/indigo/of_connection_manager.h

```diff
@@ -127,6 +131,18 @@ typedef struct indigo_cxn_params_tcp_over_ipv6_s {
     uint16_t controller_port;
 } indigo_cxn_params_tcp_over_ipv6_t;
 
+#ifdef DEPENDMODULE_INCLUDE_OFCONNECTIONMANAGER2
+/**
+ * Unix domain socket parameters
+ *     unix_path Path of unix domain socket
+ */
+#define INDIGO_CXN_UNIX_PATH_LEN 64
```

Comment:
I was actually trying to avoid a dependency on this constant, since it only seems to be defined for linux platforms.
Also if we increase the path length beyond 64 characters, MAX_CONTROLLER_DESC_LEN should also be increased.  Please let me know what you'd like to see here.